src/pages/404.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Header from '../components/Header'
+
+function NotFoundPage (): JSX.Element {
+  return (
+    <>
+      <div className='absolute w-full h-screen bg-gray-50 overflow-hidden p-2'>
+        <div className='flex items-center align-middle justify-center h-screen'>
+          <div className='text-center mt-8'>
+            <div className='flex justify-center'>
+              <div className='bg-yellow-100 italic mb-2'>
+                404
+              </div>
+            </div>
+
+            <div className='bg-white text-4xl md:text-7xl tracking-wide font-bold px-2 py-1'>
+              Page not found
+            </div>
+
+            <div className='mt-2 bg-white'>
```

Comment:
can we let leave out the background (bg-white) to let the div inherit whatever is set higher up?  

<img width="694" alt="Screen Shot 2022-05-15 at 11 11 55 AM" src="https://user-images.githubusercontent.com/3805254/168465734-ee37b13e-f760-41c6-bbd4-3d645b909bd1.png">

Reply:
Sure thing